site/content/entry/yuzu-mini-dev-2/index.md

```diff
@@ -46,27 +45,27 @@ What got me into emulation mainly was, yuzu's very [first video](https://www.you
 **Q: That's really interesting to hear. Before yuzu and Switch emulation, have you ever worked on other older emulators?**
 
 **R:** No, I've never worked on a non-Switch emulator. The closest thing I did related to emulation was a clone of Pico-8 Celeste for the NES in 6502 assembly.
-It was a fun experience because I learnt the curse and blessing that it is, working that close to the hardware.
-And it showed me what a pain it was, to program classics like Super Mario Bros. 3.
+It was a fun experience, because I learnt the curse, and blessing, that it is, working that close to the hardware; it showed me what a pain it was to program classics like Super Mario Bros. 3.
```

Comment:
Should be: It was a fun experience because I learnt the curse, and blessing, that it is to work that close to the hardware; it showed me what a pain it was to program classics like Super Mario Bros. 3.